src/tests/unit/tests/electron/flux/store/electron-window-state-store.test.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { WindowStateActions } from '../../../../../../electron/flux/action/window-state-actions';
+import { ElectronWindowStateStore } from '../../../../../../electron/flux/store/electron-window-state-store';
+import { ElectronWindowStateStoreData } from '../../../../../../electron/flux/types/electron-window-state-store-data';
+import { createStoreWithNullParams, StoreTester } from '../../../../common/store-tester';
+
+describe('ElectronWindowStateStore', () => {
+    describe('constructor', () => {
+        it('has no side effects', () => {
+            const testSubject = createStoreWithNullParams(ElectronWindowStateStore);
+            expect(testSubject).toBeDefined();
+        });
+    });
+
+    it('returns device connect as the default state', () => {
+        const testSubject = new ElectronWindowStateStore(new WindowStateActions());
+        testSubject.initialize();
+
+        expect(testSubject.getState()).toEqual(testSubject.getDefaultState());
+        expect(testSubject.getState()).toEqual(getDeviceConnectViewState());
```

Comment:
not sure what the second `expect` is doing here, is it really needed?

Reply:
we test the default state is device connect view.
& we also test that the initial test is device connect view.

it might have been more clear, if we explicitly validate both states to be same as getDeviceConnectViewState()

Reply:
I think this is a good point that we are missing on other store test (the default state) but I'd suggest using a snapshot instead.

Reply:
So should I update the expect statements to be 

expect(testSubject.getDefaultState()).toEqual(getDeviceConnectViewState());
expect(testSubject.getState()).toEqual(getDeviceConnectViewState());

Reply:
So I pushed a change with the updated expect statements before this comment; do I need to change to snapshots for this PR or add it in my next PR? @smoralesd

Reply:
added snapshots